src/components/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Navbar, Nav } from 'react-bootstrap';
```

Comment:
Don't import like this as it increases the bundle size : use import `Navbar from 'react-bootstrap/Navbar`